libraries/Microsoft.Bot.Builder.Ai.QnA/QnAMakerMiddleware.cs

```diff
@@ -29,6 +32,14 @@ public QnAMakerMiddleware(QnAMakerMiddlewareOptions options, HttpClient httpClie
                 if (!string.IsNullOrEmpty(messageActivity.Text))
                 {
                     var results = await _qnaMaker.GetAnswers(messageActivity.Text.Trim()).ConfigureAwait(false);
+                    var traceInfo = new QnAMakerTraceInfo()
+                    {
+                        QueryResults = results,
+                        QnAMakerOptions = _options
+                    };
+                    var traceActivity = Activity.CreateTraceActivity("QnAMakerMiddleware", QnAMakerTraceType, traceInfo, QnAMakerTraceLabel);
```

Comment:
Option 1: public const MyFancyNAme = "QnAMakerMiddleware"; 
Option 2: use the nameof operator